izanami-server/javascript/src/izanami/pages/FeaturesPage.js

```diff
@@ -191,11 +191,20 @@ export class FeaturesPage extends Component {
     }
   };
 
-  renderIsActive = item => <SimpleBooleanInput value={item.enabled} onChange={v => {
-    IzanamiServices.fetchFeature(item.id).then(feature => {
-      IzanamiServices.updateFeature(item.id, { ...feature, enabled: v });
-    })
-  }} />;
+  renderIsActive = item => <SimpleBooleanInput value={item.enabled} onChange={(v, input) => {
+    let confirmRes = true;
+    if (process.env.NODE_ENV === 'production') {
```

Comment:
Hi, it could be great if this option was in the configuration file. I don't want this to be enabled for everyone.